libs/widget-lib/src/types.ts

```diff
@@ -11,7 +11,7 @@ export interface EthereumProvider {
   enable(): Promise<void>
 }
 
-export type CowSwapWidgetEnv = 'local' | 'prod'
+export type CowSwapWidgetEnv = 'local' | 'prod' | 'dev' | 'vercel'
```

Comment:
I'd prefer to use `pr` rather than `vercel`

Reply:
100% agree! Renamed